LICENSE.txt

```diff
@@ -30,7 +30,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ====
 
-All files located in the node_modules and external directories are
+All files located in the node_modules and diff.js file in src folder are
```

Comment:
Let's revert this. The new `src/diff.js` is modified, not maintained externally.